sirepo/auth_db.py

```diff
@@ -27,10 +27,10 @@
 thread_lock = threading.RLock()
 
 
-def all_uids():
+def guest_uids():
     with thread_lock:
         res = set()
-        for u in UserRegistration.query.all():
+        for u in UserRegistration.query.filter_by(display_name=None):
```

Comment:
encapsulate this in `auth`, e.g. auth.guest_uids() and this function can go away. It would be a better implicit coupling with the concept that guests don't have display_names (currently only in `auth`).